touchforms/backend/xformplayer.py

```diff
@@ -202,6 +212,21 @@ def session_state(self):
         state = dict((k, v) for k, v in state.iteritems() if v is not None)
         return state
 
+    def get_xmlns(self):
+        metadata = self.fem.getForm().getMainInstance().getMetaData()
+        return metadata.get("XMLNS")
+
+    def evaluate_xpath(self, xpath):
```

Comment:
cool idea though I think using the JR framework you would need at least 
- the form definition
- the form instance
- all relevant external instances
- the case and ledger dbs
- the current question pointer (?)

Might be tricky to abstract out all of that